src/customers/customer-account-manage.md

```diff
@@ -99,6 +99,20 @@ See [Managing Subscribers]({% link marketing/newsletter-subscribers.md %}) for m
 
    To learn more, see [Updating Customer Accounts]({% link customers/customer-account-update.md %}).
 
+## Export customers accounts
+
+1. On the _Admin_ sidebar, go to **Customers** > **All Customers**.
+
+1. In the action menu, click Export and select the desired format:
```

Comment:
I don't think it should be termed "action menu" - this might get confused with the "Action" dropdown on the left of the grid. What about Table Header Menu?